cfme/tests/cloud/test_providers.py

```diff
@@ -1256,7 +1256,19 @@ def test_ec2_create_sns_topic():
             1.
             2. Check SNS topic for this region in AWS
     """
-    pass
+    # preparations for test
+    request.addfinalizer(lambda: provider.delete_if_exists())
```

Comment:
Jut note: This is enough to do the same job.
```suggestion
    request.addfinalizer(provider.delete_if_exists)
```

Reply:
Changed it.